bigtable/api/SnippetsTest/WriteTest.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) 2018 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+using System;
+using System.Linq;
+using Xunit;
+using Google.Cloud.Bigtable.Admin.V2;
+
+public class BigtableClientFixture : IDisposable
+{
+    public readonly string projectId =
+        Environment.GetEnvironmentVariable("GOOGLE_CLOUD_PROJECT");
+    public readonly string instanceId =
+        Environment.GetEnvironmentVariable("TEST_BIGTABLE_INSTANCE");
+    public readonly string tableId = $"mobile-time-series-{Guid.NewGuid().ToString().Substring(0, 8)}";
```

Comment:
Any reason to avoid using the whole guid?

Reply:
there is a limit on the length of table names in Bigtable

Reply:
also, even though we delete the table, we need to use guid because it can take a few minutes for the table to actually be deleted which can cause issues with the test when run multiple times with the same table name